wpengine/atlas-content-modeler
# [MTKA-1427] Blueprints: export categories and post_tags for WP core posts

## Description

Extends the `wp acm blueprint export` command to export category and post_tag terms for core WP post types.

https://wpengine.atlassian.net/browse/MTKA-1403
https://wpengine.atlassian.net/browse/MTKA-1427

## Testing

I updated existing tests.

### To test manually

1. Create a regular WordPress post with one or more categories and one or more tags.
2. Run `wp acm blueprint export` and inspect the generated `acm.json` file.

You should see category and post_tag terms under the 'post_terms' key.

## Screenshots

Output is unchanged:

```
> wp acm blueprint export --name="Hello World" --post-types=post --open
Collecting ACM data.
Collecting ACM models.
Collecting ACM taxonomies.
Collecting posts.
Collecting post tags.
Collecting terms.
Collecting post meta.
Collecting media.
Collecting ACM relationships
Writing acm.json manifest.
Generating zip.
Opening blueprint temp folder.
Success: Blueprint saved to /var/folders/mc/yf37tb4j5bjbwhv6z0m1bcfm0000gp/T/acm/hello-world/hello-world.zip.
```

## Documentation Changes

Docs have been updated.

## Dependant PRs

Targets `blueprints/core-taxonomies` so that work to _import_ category and post_tag terms can target the same branch. Both features should be merged to main together.
